packages/server/lib/util/git.ts

```diff
@@ -0,0 +1,84 @@
+import execa from 'execa'
+import path from 'path'
+import os from 'os'
+import Debug from 'debug'
+import type { GitInfo } from '@packages/types'
+
+const debug = Debug('cypress:server:git')
+
+// matches <timestamp> <when> <author>
+// $ git log -1 --pretty=format:%ci %ar %an <file>
+// eg '2021-09-14 13:43:19 +1000 2 days ago Lachlan Miller
+const GIT_LOG_REGEXP = /(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [-+].+?)\s(.+ago)\s(.*)/
+const GIT_LOG_COMMAND = `git log -1 --pretty="format:%ci %ar %an"`
+
+const getInfoWindows = async (absolutePaths: string[]) => {
```

Comment:
This feels a bit dirty but it's much faster than the other solutions I found. This one basic runs instantly. There's probably a cleaner way, but I am not sure what that is. 

This code was meticulously tested on a **real** windows machine. I executed it using node.js. I don't have a windows machine with a Cypress development env, so I couldn't fully test it there, but I will look into this.